pnpm-lock.yaml

```diff
@@ -1,4 +1,4 @@
-lockfileVersion: '6.0'
+lockfileVersion: '6.1'
```

Comment:
are you using the latest pnpm? I think there was a weird bug in a version of pnpm that did this, and was reverted. they may have un-reverted it though so if you're on the latest pnpm I'm sure you're good! I'll check too...

Reply:
@travis I'm on pnpm 8.6.1 (i just got latest) and I did a re- `pnpm install` and did not produce different lockfile

Reply:
(i upgraded to 8.12.0 and it produces identical lockfile)